controllers/feeds/getFeedChat.js

```diff
@@ -79,6 +80,7 @@ const getDetail = (req, b, id) => {
     isAnonym,
     isOwnSignedPost,
     isOwnPost,
+    isOwnAnonymousPost,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variable `isOwnAnonymousPost` is now included in the returned object. This seems to be consistent with the PR's purpose of distinguishing between anonymous and signed posts.
```diff
+ isOwnAnonymousPost,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->